### PR TITLE
native: perform initialisation for the set of hardforks

### LINKED ISSUE #3433: Port natives initialization improvement
https://github.com/neo-project/neo/issues/3200.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #3433.

### CODE REVIEW

File: pkg/core/native/management.go [hunk 1]
```diff
@@ -654,9 +657,21 @@ func (m *Management) OnPersist(ic *interop.Context) error {
 		if err != nil {
 			return fmt.Errorf("failed to put contract state: %w", err)
 		}
-		if err := native.Initialize(ic, activeIn, hfSpecificMD); err != nil {
-			return fmt.Errorf("initializing %s native contract at HF %d: %w", md.Name, activeIn, err)
+
+		// Deploy hardfok (contract's ActiveIn) is not a part of contract's active hardforks and
```
Comment (roman-khimov) [inline pkg/core/native/management.go:661]: hardfork

**Review by roman-khimov — COMMENTED** [review]
Missing tests for it.

**Comment by AnnaShaleva** [comment]
> Missing tests for it.

I just don't know how to test it, we need a native contract to check that initialization works as expected, but we don't have such native contracts that are being initialized in non-ActiveIn hardfork.

File: pkg/core/native/management.go (hunk 1 above)
Comment (AnnaShaleva) [reply pkg/core/native/management.go]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**